.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug Report
+description: Report incorrect behavior in the HydroMT-SFINCS library
+labels: [Bug, Needs refinement]
```

Comment:
Not sure this is an issue but I don't think we have a "Needs refinement" label yet. To be sure we should make the issues we mention here at [this link](https://github.com/Deltares/hydromt_sfincs/labels). Idem for other templates

Reply:
Done!